src/constants/index.ts

```diff
@@ -2,6 +2,8 @@ import { ChainId, Exchange, JSBI, Percent, Token, WETH } from '@alchemistcoin/sd
 import { AbstractConnector } from '@web3-react/abstract-connector'
 import { injected, ledger, walletconnect } from '../connectors'
 
+export const APP_VERSION = '0.1.5'
```

Comment:
We should be able to import the version directly from package.json so it doesn't have to be manually updated here. Security concerns are minimal as the package.json is already exposed publicly

Reply:
Yea good call